maven-scm-plugin/pom.xml

```diff
@@ -228,7 +228,7 @@
     <plugins>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-plugin-plugin</artifactId>
+        <artifactId>maven-plugin-report-plugin</artifactId>
```

Comment:
I am not sure why there is no profile `reporting` in this project. IMHO this belongs there. WDYT @michael-o

Reply:
For me is ok ... even more I would be happy to remove reporting profile at all everywhere.

We always should execute all reporting plugins not only in special profile

Reply:
@kwin I move this to a profile for consistency reasons with others.

Reply:
> For me is ok ... even more I would be happy to remove reporting profile at all everywhere.
> 
> We always should execute all reporting plugins not only in special profile

This is a separate discussion. The reason is likely computational overhead.

Reply:
I moved to a profile in https://github.com/apache/maven-scm/pull/216/commits/bacee6d36cd28f2a02eedaa726e6f4a21ed75147